src/main/java/org/embulk/output/GcsOutputPlugin.java

```diff
@@ -419,6 +419,12 @@ private String getLocalMd5hash(String filePath) throws IOException
         }
     }
 
+    private static String generateRemotePath(String pathPrefix, String sequenceFormat, int taskIndex, int fileIndex, String pathSuffix)
+    {
+        String path = pathPrefix + String.format(sequenceFormat, taskIndex, fileIndex) + pathSuffix;
+        return path.replaceFirst("\\.*/*", "");
```

Comment:
> This may replace ......///foo/bar as well.

I think it's ok to in this case.

> This may replace foo/bar/./baz.

I fixed regular expression to keep in this case 8387504

Although, Google doesn't allows to use `.` as path string, I think it's ok to use following rules.

* plugin normalize only when `.` or `/` exists at beginning.
* User responsible for other cases